prod-aws/kafka-shared-msk/cbc/cbc.tf

```diff
@@ -52,11 +44,3 @@ module "cbc_events_indexer" {
   consume_groups   = ["cbc.cbc-events-indexer"]
   cert_common_name = "cbc/cbc-events-indexer"
 }
-
-module "cbc_account_events_relay" {
-  source           = "../../../modules/tls-app"
-  produce_topics   = []
-  consume_topics   = ["account-identity.legacy.account.events"]
-  consume_groups   = ["account-identity.cbc-account-events-relay-v2"]
-  cert_common_name = "cbc/cbc-account-events-relay-v2"
-}
```

Comment:
In `dev` this module produced to a topic defined in the parent module https://github.com/utilitywarehouse/kafka-cluster-config/commit/cd7c0f097ab7fc6cedd48a93a70a9ccae27ee7e2#diff-0a54529face97511bb22c07a59a75a17323f77b7238c641b4f27abd89a7fbc14R1300 since it doesn't here we can just delete the entire thing.